src/lib/blueprint.ts

```diff
@@ -64,6 +64,12 @@ export interface Resource {
   undocumentedMessage: string
   isDraft: boolean
   draftMessage: string
+  propertyGroups: PropertyGroup[]
```

Comment:
It should be easier if we make this
```suggestion
  propertyGroups: Record<string, PropertyGroup>
```

Reply:
3d4519884f307a9364892e83f5a4d883da9107f4